compiler/rustc_ast/src/visit.rs

```diff
@@ -636,6 +639,15 @@ pub fn walk_generics<'a, V: Visitor<'a>>(visitor: &mut V, generics: &'a Generics
     walk_list!(visitor, visit_where_predicate, &generics.where_clause.predicates);
 }
 
+pub fn walk_closure_binder<'a, V: Visitor<'a>>(visitor: &mut V, binder: &'a ClosureBinder) {
+    match binder {
+        ClosureBinder::NotPresent => {}
+        ClosureBinder::For { generic_params, .. } => {
```

Comment:
Could you use an exhaustive pattern, and explicitly ignore field?  This will catch future modifications to the enum.

Reply:
Done in abcd3f58537c3ed5e0e56cff6948eb35fd317ac1